@xen-orchestra/proxy/src/app/mixins/backups/index.js

```diff
@@ -154,17 +154,36 @@ export default class Backups {
           },
         ],
         importVmBackup: [
-          defer(($defer, { backupId, remote, srUuid, xapi: xapiOpts }) =>
-            using(this.getAdapter(remote), this.getXapi(xapiOpts), async (adapter, xapi) =>
-              new ImportVmBackup({ adapter, backupId, srUuid, xapi }).run()
-            )
+          defer(($defer, { backupId, remote, srUuid, streamLogs = false, xapi: xapiOpts }) =>
+            using(this.getAdapter(remote), this.getXapi(xapiOpts), async (adapter, xapi) => {
+              const metadata = await adapter.readVmBackupMetadata(backupId)
```

Comment:
Why did you choose to move this out?

Reply:
Some metadata info are needed in the restore task.